src/scrot.c

```diff
@@ -860,6 +864,38 @@ static Imlib_Image scrotGrabShotMulti(void)
     return stalkImageConcat(&images, HORIZONTAL);
 }
 
+static Imlib_Image scrotGrabShotMonitor(void)
+{
+    int eventBase, errBase;
+
+    if (!XineramaQueryExtension(disp, &eventBase, &errBase))
+        err(EXIT_FAILURE, "Xinerama extension not found");
+
+    int numScreens = 0;
+    XineramaScreenInfo* screens = XineramaQueryScreens(disp, &numScreens);
```

Comment:
TODO:

- [ ] Some of the code is pre-style WebKit. [Put the * next to the variable name and not the type](https://webkit.org/code-style-guidelines/#pointers-non-cpp)(`screens` and `mon`)
- [ ] Add your `Copyright` in the .h and .c files that you have changed.

Reply:
Sure, now done.